index.js

```diff
@@ -127,3 +127,8 @@ Math.pow = function (n) {
 
   return str;
 };
+
+Math.random = function(){
```

Comment:
Nitpick: can you change this to `Math.random = function() {` (space after the `()` to match the formatting of the other functions)